quora-api/src/main/java/com/upgrad/quora/api/controller/AnswerController.java

```diff
@@ -29,6 +29,71 @@
 
     @Autowired
     private AnswerService answerService;
+	
+	@RequestMapping(
```

Comment:
Use PostMapping annotation instead of RequestMapping .. This will led to more readable code